joinwiz_core/src/main/scala/joinwiz/testkit/SeqJoinImpl.scala

```diff
@@ -21,4 +21,9 @@ class SeqJoinImpl[L, R](op: Expr[L, R], left: Seq[L], right: Seq[R]) {
       b <- right
       if op(a, b)
     } yield (a, b)
+
+  def leftAntiJoin(): Seq[L] = {
+    leftJoin().collect { case e if e._2.isEmpty => e._1 }
```

Comment:
still can be a bit nicer
```scala
.collect { case (left, None) => left}
```

what do you think?

Reply:
changed